crates/cxx-qt-gen/src/generator/cpp/property/mod.rs

```diff
@@ -80,8 +81,37 @@ mod tests {
     use indoc::indoc;
     use pretty_assertions::assert_str_eq;
     use quote::format_ident;
-    use syn::{parse_quote, ItemMod, ItemStruct};
+    use syn::{parse_quote, Error, ItemMod, ItemStruct};
+
+    pub fn require_pair(fragment: &CppFragment) -> Result<(String, String)> {
+        match fragment {
+            CppFragment::Pair { header, source } => Ok((header.clone(), source.clone())),
+            _ => {
+                let fragment_string: String = fragment.clone().into();
+                Err(Error::new_spanned(fragment_string, "Expected Pair!"))
```

Comment:
Have you tested what happens when the error occurs? Eg do the tokens even create a nice error message?

As I would note that here we are in C++ land so you don't have to return a `syn::Error` you could return a normal error type or panic / unwrap in the test :thinking:

Reply:
But maybe a `syn::Error` is able to create a nice message here?

Reply:
panic!() isn't a good option because it will be an uncovered line, but yeah I was only using the tokens to create a syn error so can maybe switch to a regular result and avoid needing it tbh